### PR TITLE
 Remove unnecessary TODO comment

### PR DESCRIPTION
**Pull request by bradengroom**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Comment by bradengroom** [comment]
I've signed the CLA, but the tool indicates that I haven't for some reason.

File: Lib/test/test_asyncio/test_events.py [hunk 1]
```diff
@@ -2351,33 +2351,32 @@ def callback(*args):
...
+        self.assertGreaterEqual(h1, h2)
+        self.assertGreaterEqual(h2, h1)
+        self.assertLessEqual(h1, h2)
+        self.assertLessEqual(h2, h1)
+        self.assertLessEqual(h1, h2)
+        self.assertLessEqual(h2, h1)
+        self.assertGreaterEqual(h1, h2)
+        self.assertGreaterEqual(h2, h1)
+        self.assertEqual(h1, h2)
+        self.assertEqual(h1, h2)
```
Comment (srinivasreddy) [inline Lib/test/test_asyncio/test_events.py]: this is dupe of above test case , can safely removed.

**Comment by srinivasreddy** [comment]
Though OP has put a TODO to write them using Lessthan and Greaterthan methods. I think it is better to leave them as is. I do not see value in re writing them.

**Comment by bradengroom** [comment]
It feels like a lot of these assertions overlap now that I look more closely. For example, should
```
self.assertFalse(h1 < h2)
self.assertFalse(h2 < h1)
```

Just be ?:
```
self.assertEqual(h1, h2)
```

**Comment by bradengroom** [comment]
@srinivasreddy It seems like a lot of these assertions can be removed. Is it necessary to compare every combination of less than, greater than, and equality? Can't these assertions be simplified to just this?
https://github.com/python/cpython/pull/9876/commits/1c9e665312674a9ad72b3b8b004f354a7966f14c

**Comment by srinivasreddy** [comment]
Nope, I guess. TimerHandle can also give you float seconds.

**Comment by serhiy-storchaka** [comment]
Is not the purpose of this test testing comparison operators?

**Comment by bradengroom** [comment]
@serhiy-storchaka You're right. Thanks. I guess there is value in these. I'll just remove the `TODO` since some of these cannot even be written using the suggested functions. There is no `assertNotLess`, and we can't just call `assertGreater` since that tests a different path.

**Comment by bradengroom** [comment]
I could rewrite some of these where appropriate functions exist, but there's not great value in this as @srinivasreddy pointed out. So I think we can just remove the TODO since there's no reason to change it.

**Comment by csabella** [comment]
@bradengroom, thank you for the suggestion.  Your comment

> There is no assertNotLess, and we can't just call assertGreater since that tests a different path.

isn't exactly true because there is an `assertLess` and an `assertLessEqual`, so all the test cases can be covered by other asserts.

I would suggest opening a ticket on the bug tracker mentioning the TODO and then letting the asnycio team decide if it's something that should be changed.  They may consider it code churn and not worth it, or they may want to change the asserts.  For now, I'm going to close this pull request.

**Pull request closed without merging**